### PR TITLE
DDCE-5602 Trusts IV- We could not confirm your identity

### PR DESCRIPTION
**Pull request by matthew-gibson-18**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -110,3 +110,15 @@ notFound.p3 = Os nad oes gennych gyfeirnod unigryw ar gyfer yr ymddiriedolaeth,
...
+couldNotConfirmIdentity.title = Nid oedd modd i ni gadarnhau pwy ydych
+couldNotConfirmIdentity.heading = Nid oedd modd i ni gadarnhau pwy ydych
+couldNotConfirmIdentity.p1 = Mae angen i ni siarad â chi i’ch helpu i gael mynediad at yr ymddiriedolaeth.
+couldNotConfirmIdentity.p2 = Cysylltwch â Gwasanaeth Cwsmeriaid Cymraeg CThEF
+couldNotConfirmIdentity.p3 = Ffôn:
+couldNotConfirmIdentity.p4 = 0300 200 1900
+couldNotConfirmIdentity.p5 = O’r tu allan i’r DU:
+couldNotConfirmIdentity.p6 = +44 300 200 1900
+couldNotConfirmIdentity.p7 = Oriau agor:
+couldNotConfirmIdentity.p8 = Dydd Llun i ddydd Gwener: 08:30 - 17:00
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply conf/messages.cy:123]: Looking at the Welsh screenshots this is probably not a issue, the phone number is different so maybe the operating times are also different.

**Review by MRBeresford — DISMISSED** [review]
Looks good to me, just one really really minor comment

File: conf/messages [hunk 2]
```diff
@@ -112,4 +112,16 @@ notFound.li.2 = a Unique Reference Number (URN) which has 15 numbers and charact
...
+couldNotConfirmIdentity.title = We could not confirm your identity
+couldNotConfirmIdentity.heading = We could not confirm your identity
+couldNotConfirmIdentity.p1 = We need to speak to you to help you to access the trust.
+couldNotConfirmIdentity.p2 = Contact the trusts helpline
+couldNotConfirmIdentity.p3 = Telephone:
+couldNotConfirmIdentity.p4 = 0300 123 1072
+couldNotConfirmIdentity.p5 = Outside UK:
+couldNotConfirmIdentity.p6 = +44 304 123 1472
+couldNotConfirmIdentity.p7 = Opening times:
+couldNotConfirmIdentity.p8 = Monday to Friday: 9am to 5 pm
```
[the remark this answers is not in the document]
Comment (matthew-gibson-18) [reply conf/messages]: taken the space out

**Review by MRBeresford — APPROVED** [review]
LGTM

**Pull request merged**